src/app/components/TodoList.js

```diff
@@ -0,0 +1,119 @@
+import TodoApi from '../services/TodoApi';
+import SelectColor from './SelectColor';
+
+var React = require('react');
+var ReactDOM = require('react-dom');
+
+export default class TodoList extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      error: null,
+      isLoaded: false,
+      items: []
+    }
+  }
+  componentDidMount() {
+    this.reloadList()
+  }
+
+  reloadList(){
+    new TodoApi().readFetch().then(
+      (result) => {
+        this.setState({
+          isLoaded: true,
+          items: result.items
+        });
+      },
+      (error) => {
+        this.setState({
+          isLoaded: true,
+          error
+        });
+      }
+    )
+  }
+
+  readNewItem(){
+    let newItem = document.getElementById('todo')
+    let newItemText = newItem.value
+    newItem.value = ''
+    let newItemColor = document.getElementById('addColor').value
+    let newItemCheck = false
+    let item = {
+      text: newItemText,
+      color: newItemColor,
+      check: newItemCheck
+    }
+    return item
+  }
+
+  createNewItem(e){
+    new TodoApi().createFetch(e).then(response => {
+      this.reloadList()
+      console.log(response)
+    })
+  }
+
+  updateItem(itemText, itemCheck, itemId, itemColor){
+   new TodoApi().updateFetch(itemText, itemCheck, itemId, itemColor).then(response => {
+      this.reloadList()
+      console.log(response)
+    })
+  }
+
+  deleteItem(id){
+    new TodoApi().deleteFetch(id).then(response => {
+      this.reloadList()
+      console.log(response)
+    })
+  }
+  
+
+  renderList() {
+    const { error, isLoaded, items } = this.state;
+
+    if (error) {
+      return <div>Помилка: {error.message}</div>
+    } else if (!isLoaded) {
+      return <div>Завантаження...</div>
+    } else {
+      return (
+        <ul id='todolist'>
```

Comment:
try to use classes instead of ids

Reply:
Hi, thank you for code review😊
It helped me a lot. 
I am going to do refactoring soon